charts/datadog/values.yaml

```diff
@@ -327,6 +327,11 @@ datadog:
     # datadog.systemProbe.conntrackMaxStateSize -- the maximum size of the userspace conntrack cache
     conntrackMaxStateSize: 131072  # 2 * maxTrackedConnections by default, per  https://github.com/DataDog/datadog-agent/blob/d1c5de31e1bba72dfac459aed5ff9562c3fdcc20/pkg/process/config/config.go#L229
 
+    # datadog.systemProbe.enableRuntimeCompiler -- Enable the runtime compiler for eBPF probes
+    enableRuntimeCompiler: false
```

Comment:
Is it worth enforcing a minimum agent version for this?

We do something similar here: https://github.com/DataDog/helm-charts/blob/master/charts/datadog/templates/system-probe-configmap.yaml#L7

Reply:
It feels unnecessary to me to add version checking for every new config option. I understand the `networkMonitoring` one because that determines whether the system-probe even runs or not. This (for now) is still an opt-in feature.